components/eamxx/tpls/CMakeLists.txt

```diff
@@ -44,6 +44,10 @@ if (SCREAM_ENABLE_MAM)
     -DCMAKE_INSTALL_PREFIX=${HAERO_INSTALL_PREFIX}
     -DCMAKE_BUILD_TYPE=${mam_build_type}
     -DCMAKE_VERBOSE_MAKEFILE=${CMAKE_VERBOSE_MAKEFILE}
+    -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
+    -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+    -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+    -DCMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER}
```

Comment:
Haero has no Fortran code, so we don't need to pass `CMAKE_Fortran_COMPILER` here.